lib/countryParser.py

```diff
@@ -0,0 +1,49 @@
+from lxml import etree
+
+from helpers.logHelpers import createLog
+from helpers.errorHelpers import ProcessingError
+
+logger = createLog('country_code_generator')
+
+
+def loadCountryCodes():
```

Comment:
Might be worth just turning this list of country codes into a module.

Reply:
I'm not sure if this will end up being reused (most places use the ISO country codes, not the MARC ones), but if it is then I probably will turn it into a module (especially since one doesn't seem to exist at all)